src/main/java/io/github/debuggyteam/architecture_extensions/resource/ResourceUtils.java

```diff
@@ -50,7 +50,7 @@ private static Cache loadOrGetCache(ModContainer resourceOwner, ResourceType res
 			return null;
 		});
 	}
-
+	
```

Comment:
Why is there just floating whitespace here?

Reply:
editorconfig thing